service-api/src/main/java/greencity/dto/AddBonusesToUserDto.java

```diff
@@ -0,0 +1,15 @@
+package greencity.dto;
+
+import lombok.*;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
```

Comment:
use only @Builder or @Setter, but don`t use them together

Reply:
and also  refactoring code smell